src/test/java/io/github/vlsergey/springdatarestutils/ToOpenApiActionImplTest.java

```diff
@@ -36,9 +40,10 @@ class ToOpenApiActionImplTest {
     private TaskProperties taskProperties;
 
     static void assertEquals(URL expectedUrl, File actualFile) throws IOException {
+	Yaml yaml = new Yaml(new Constructor(Map.class));
 	String expected = Resources.toString(expectedUrl, StandardCharsets.UTF_8).replace("\r", "");
 	String actual = new String(Files.readAllBytes(actualFile.toPath()), StandardCharsets.UTF_8).replace("\r", "");
-	Assertions.assertEquals(expected, actual);
+	Assertions.assertEquals(Optional.ofNullable(yaml.load(expected)), Optional.ofNullable(yaml.load(actual)));
```

Comment:
Why is `Optional.ofNullable` needed in this test but not in other tests? You probably want to justify its usage.

Reply:
Changed to explicit type casting.

Reply:
My question is actually:

In https://github.com/ThugJudy/spring-data-rest-utils/pull/1, you used `assertEquals` with `yaml.load` without the need to cast. Why do you need to the cast here? What's the difference between these two PRs that makes the cast necessary?

Reply:
The reason for this is that, we are typecasting it into map object in the previous PR. The return data type is ambiguous hence we have to explicitly specify it. An alternate approach to maintain uniformity would be to replace line 39 and 40 with the following lines:
```
Map<String, Object> expected = yaml.load(Resources.toString(expectedUrl, StandardCharsets.UTF_8).replace("\r", ""));
Map<String, Object> actual = yaml.load(new String(Files.readAllBytes(actualFile.toPath()), StandardCharsets.UTF_8).replace("\r", ""));
```

Reply:
Makes sense. Keeping the cast inline looks better.